packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -82,28 +83,23 @@ class WalletsController {
     mnemonic: string
     receiveAddressNumber: number
     changeAddressNumber: number
```

Comment:
just have a question: why does import keystore require receive address number and change address number?

Reply:
@duanyytop

Reply:
Creating and importing wallet will provide 20(maybe any number) addresses for user. Please reference Neuron Addresses UI.

Reply:
The functionality of them is clear, but what add them into the keystore? I mean, is the keystore related to providing 20 addresses?

Reply:
They're parallel in structure